internal/installcfg/installcfg.go

```diff
@@ -134,185 +143,132 @@ func getMachineCIDR(cluster *models.Cluster) (string, error) {
 	return "", errors.New(errStr)
 }
 
+func getBasicInstallConfig(cluster *models.Cluster, machineCIDR string) *InstallerConfigBaremetal {
+	return &InstallerConfigBaremetal{
+		APIVersion: "v1",
+		BaseDomain: cluster.BaseDNSDomain,
+		Networking: struct {
+			NetworkType    string `yaml:"networkType"`
+			ClusterNetwork []struct {
+				Cidr       string `yaml:"cidr"`
+				HostPrefix int    `yaml:"hostPrefix"`
+			} `yaml:"clusterNetwork"`
+			MachineNetwork []struct {
+				Cidr string `yaml:"cidr"`
+			} `yaml:"machineNetwork"`
+			ServiceNetwork []string `yaml:"serviceNetwork"`
+		}{
+			NetworkType: "OpenShiftSDN",
+			ClusterNetwork: []struct {
+				Cidr       string `yaml:"cidr"`
+				HostPrefix int    `yaml:"hostPrefix"`
+			}{
+				{Cidr: cluster.ClusterNetworkCidr, HostPrefix: int(cluster.ClusterNetworkHostPrefix)},
+			},
+			MachineNetwork: []struct {
+				Cidr string `yaml:"cidr"`
+			}{
+				{Cidr: machineCIDR},
+			},
+			ServiceNetwork: []string{cluster.ServiceNetworkCidr},
+		},
+		Metadata: struct {
+			Name string `yaml:"name"`
+		}{
+			Name: cluster.Name,
+		},
+		Compute: []struct {
+			Name     string `yaml:"name"`
+			Replicas int    `yaml:"replicas"`
+		}{
+			{Name: "worker", Replicas: countHostsByRole(cluster, "worker")},
+		},
+		ControlPlane: struct {
+			Name     string `yaml:"name"`
+			Replicas int    `yaml:"replicas"`
+		}{
+			Name:     "master",
+			Replicas: countHostsByRole(cluster, "master"),
+		},
+		PullSecret: cluster.PullSecret,
+		SSHKey:     cluster.SSHPublicKey,
+	}
+}
+
+// [TODO] - remove once we decide to use specific values from the hosts of the cluster
+func getDummyMAC(dummyMAC string, count int) (string, error) {
+	hwMac, err := net.ParseMAC(dummyMAC)
+	if err != nil {
+		logrus.Warn("Failed to parse dummyMac")
+		return "", err
+	}
+	hwMac[len(hwMac)-1] = hwMac[len(hwMac)-1] + byte(count)
+	return hwMac.String(), nil
+}
+
+func setPlatformInstallconfig(cluster *models.Cluster, cfg *InstallerConfigBaremetal) error {
+	// set hosts
+	numMasters := countHostsByRole(cluster, "master")
+	numWorkers := countHostsByRole(cluster, "worker")
+	masterCount := 0
+	workerCount := 0
+	hosts := make([]host, numWorkers+numMasters)
+
+	// dummy MAC and port, once we start using real BMH, those values should be set from cluster
+	dummyMAC := "00:aa:39:b3:51:10"
+	dummyPort := 6230
+
+	for i := range hosts {
+		logrus.Infof("Setting master, host %d, master count %d", i, masterCount)
```

Comment:
```suggestion
		logrus.Infof("Setting host info, host %d, master count %d", i, masterCount)
```
You are setting both workers and masters

Reply:
as explained above, this is a debug log, not needed anymore, i removed it